lib/resources/r4/related_person_patch.yaml

```diff
@@ -159,3 +160,25 @@ operations:
         <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `family` is intended to be replaced.</li>
         <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
       </ul>
+
+  - name: replace-name-given
+    path: /name/{index}/given
+    operation: replace
+    type: String
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#HumanName.given
+    description: Replace the `HumanName.given` of the name at the given <code>{index}</code> in the list of related person names.
+    example: |
+      {
+        "path": "/name/0/given",
+        "op": "replace",
+        "value": [
+          "Harry",
+          "James"
+        ]
+      }
+    note: >
```

Comment:
We included this note for create. we might need to include it here as well
````
Up to two given names may be provided. If additional given names are required they must be provided in the second value as a single string separated by blank spaces.
````

Reply:
done https://github.com/cerner/fhir.cerner.com/commit/71c3d46a0e8945dfd2a7b0e3a3d9ad9a90c62bb8